software/runtime/runtime.mk

```diff
@@ -134,6 +134,9 @@ RISCV_CCFLAGS_TESTS ?= $(RISCV_FLAGS_GCC) $(RISCV_FLAGS_COMMON_TESTS) -fvisibili
 %.h: %.args
 	cat $< | xargs $(python) $(MEMPOOL_DIR)/scripts/gen_data.py --clangformat=$(LLVM_INSTALL_DIR)/bin/clang-format -o $@
 
+%.h: %.py
+	$(python) $*.py
```

Comment:
Here you could also use `$(python) $<` since that stands for the first dependency. Just to say, but this solution works just as well.

Reply:
Thanks Samuel, I updated the Changelog and corrected these last two errors